d2/src/main/java/com/linkedin/d2/xds/XdsChannelFactory.java

```diff
@@ -31,10 +32,34 @@ public class XdsChannelFactory
 
   private final SslContext _sslContext;
   private final String _xdsServerUri;
+  private final boolean _forceRandomIp;
```

Comment:
Can this be made more generic to accept the load balancing policy name as a param?

Reply:
ACK

Reply:
Done